src/components/modals/ButtonWithModal.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+
+import { Button, ButtonProps } from '../buttons/Button';
+
+import {
+  ModalProps,
+  ModalWithActionButtons,
+  ActionButton,
+} from './ModalWithActionButtons';
+
+export interface ButtonWithModalProps extends ButtonProps {
+  modalTitle: string;
```

Comment:
i thought we were gonna embed modalProps

Reply:
Yea